### PR TITLE
Hello rails back-end 

### PR DESCRIPTION
**Pull request by mike2611**
## Changes 
- Add a Postgres database set up.
- Create a table for storing your greetings.
- Create an API endpoint that selects a random greeting from the table

[React front-end PR](https://github.com/mike2611/hello-react-front-end/pull/2)

### CODE REVIEW

File: app/controllers/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def index
+    id = rand(1..Greeting.count)
+    render json: Greeting.find(id)
```
Comment (SajjadAhmad14) [inline app/controllers/greetings_controller.rb:4]: React app does not show a random greeting. Kindly make sure that all greetings are randomly rendered when an API GET request is received.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Miki, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/controllers/greetings_controller.rb (hunk 1 above)
Comment (mike2611) [reply app/controllers/greetings_controller.rb:4]: Hi @SajjadAhmad14 sorry I don't understand your requirement. I'm following the project requirements `Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).` selecting a random greeting for each API GET request.

**Comment by AshakaE** [comment]
### Hi @mike2611 🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**